internal/coroutine/coro.go

```diff
@@ -12,7 +12,6 @@ import (
 	"unsafe"
 
 	"github.com/goplus/spx/internal/debug"
-	"github.com/goplus/spx/internal/engine/platform"
```

Comment:
I'm not sure if I understand correctly, changes in this PR seems to be unrelated to the topic "Fix web value overwriting during recursive calls"?